test/runtime/connectivity.go

```diff
@@ -0,0 +1,273 @@
+		serverData := docker.ContainerInspect("server")
+		serverIP, err := serverData.Filter("{[0].NetworkSettings.Networks.cilium-net.IPAddress}")
+		Expect(err).Should(BeNil())
+		serverIPv6, err := serverData.Filter("{[0].NetworkSettings.Networks.cilium-net.GlobalIPv6Address}")
+		Expect(err).Should(BeNil())
+
+		By("Client can ping to server IPV6")
+		res := docker.ContainerExec("client", fmt.Sprintf("ping6 -c 4 %s", serverIPv6))
+		Expect(res.Correct()).Should(BeTrue())
+
+		By("Client can ping to server Ipv4")
+		res = docker.ContainerExec("client", fmt.Sprintf("ping -c 5 %s", serverIP))
+		Expect(res.Correct()).Should(BeTrue())
+
+		By("Netperf to server from client IPv6")
+		cmd := fmt.Sprintf(
+			"netperf -c -C -t TCP_SENDFILE -H %s", serverIPv6)
+		res = docker.ContainerExec("client", cmd)
+		Expect(res.Correct()).Should(BeTrue())
+	}, 300)
+
+	It("Test containers connectivity WITH policy", func() {
+		policyID, _ := cilium.PolicyImport(
+			fmt.Sprintf("%s/test.policy", cilium.ManifestsPath()), 150)
+		logger.Debug("New policy created with id '%d'", policyID)
+
+		serverData := docker.ContainerInspect("server")
+		serverIP, err := serverData.Filter("{[0].NetworkSettings.Networks.cilium-net.IPAddress}")
+		Expect(err).Should(BeNil())
+		serverIPv6, err := serverData.Filter("{[0].NetworkSettings.Networks.cilium-net.GlobalIPv6Address}")
+		Expect(err).Should(BeNil())
+
+		By("Client can ping to server IPV6")
+		res := docker.ContainerExec("client", fmt.Sprintf("ping6 -c 4 %s", serverIPv6))
+		Expect(res.Correct()).Should(BeTrue())
+
+		By("Client can ping to server Ipv4")
+		res = docker.ContainerExec("client", fmt.Sprintf("ping -c 5 %s", serverIP))
+		Expect(res.Correct()).Should(BeTrue())
+
+		By("Netperf to server from client IPv6")
+		cmd := fmt.Sprintf(
+			"netperf -c -C -t TCP_SENDFILE -H %s", serverIPv6)
+		res = docker.ContainerExec("client", cmd)
+		Expect(res.Correct()).Should(BeTrue())
+
+		By("Ping from host to server")
+		ping := docker.Node.Execute(fmt.Sprintf("ping -c 4 %s", serverIP), nil, nil)
+		Expect(ping).Should(BeTrue())
+	}, 300)
+
+	It("Test containers NAT46 connectivity ", func() {
+
+		endpoints, err := cilium.GetEndpointsIds()
+		Expect(err).Should(BeNil(), "Couldn't get endpoints IDS")
+
+		server, err := docker.ContainerInspectNet("server")
+		Expect(err).Should(BeNil())
+
+		client, err := docker.ContainerInspectNet("client")
+		Expect(err).Should(BeNil())
+
+		status := cilium.EndpointSetConfig(endpoints["client"], "NAT46", "Enabled")
+		Expect(status).Should(BeTrue())
+
+		res := docker.ContainerExec("client", fmt.Sprintf(
+			"ping6 -c 4 ::FFFF:%s", server["IPv4"]))
+		Expect(res.Correct()).Should(BeTrue())
+
+		res = docker.ContainerExec("server", fmt.Sprintf(
+			"ping6 -c 4 ::FFFF:%s", client["IPv4"]))
+		Expect(res.Correct()).Should(BeFalse(), "Unexpected NAT46 access")
+	})
+})
+
+var _ = Describe("RunConntrackTest", func() {
+
+	var initilized bool
+	var networkName string = "cilium-net"
```

Comment:
As mentioned previously, can these be moved to a separate file containing constants?